Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN apk upgrade --no-cache \
     uwsgi-python3 \
     brotli \
  && pip3 install --upgrade pip wheel setuptools \
- && pip3 install --no-cache  --no-binary :all: -r requirements.txt \
+ && pip3 install --no-cache -r requirements.txt \
```

Comment:
I think we can remove the `--no-cache`, BTW here is what we do in the SearXNG installation script ... instead of installing `requirements.txt` we install the SearXNG repository into SEARX_PYENV (`${SERVICE_HOME}/searx-pyenv`)

**for installation**

https://github.com/searxng/searxng/blob/4da1e0026c930686e8c5b0545b2c97f8b834289b/utils/searx.sh#L722-L729

**for update**

https://github.com/searxng/searxng/blob/4da1e0026c930686e8c5b0545b2c97f8b834289b/utils/searx.sh#L410-L414

Reply:
AFIK `--no-cache` will cache the dependencies in `~/.cache/pip`, but the docker image will never this directory again (since the dependencies are not updated in one docker image)